<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>971</width>
      <height>671</height>
     </rect>
    </property>
    <widget class="QWidget" name="signupScreen">
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ArcaneCrypt</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>971</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Looks like you haven't created an account yet, you can do that now</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInput">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>310</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        font-family: Segoe UI;
        font-size: 9pt;
        border-radius: 4px;
    }
    
    QLineEdit {
        background-color: #ffffff;
    }
    
    QLineEdit {
        color: #000000;
    }
    
    QLineEdit {
        border-style: solid;
        border-width: 1px;
        border-color: #c9c9c9;
    }
    
    QLineEdit {
        padding-left: 4px;
        padding-right: 4px;
    }

    QLineEdit:hover, QLineEdit:focus {
        background-color: #f2f2f2;
    }

    QLineEdit::selection {
        background-color: #0078d7;
        color: #ffffff;
    }</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string extracomment="password"/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="checkPasswordInput">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>380</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        font-family: Segoe UI;
        font-size: 9pt;
        border-radius: 4px;
    }
    
    QLineEdit {
        background-color: #ffffff;
    }
    
    QLineEdit {
        color: #000000;
    }
    
    QLineEdit {
        border-style: solid;
        border-width: 1px;
        border-color: #c9c9c9;
    }
    
    QLineEdit {
        padding-left: 4px;
        padding-right: 4px;
    }

    QLineEdit:hover, QLineEdit:focus {
        background-color: #f2f2f2;
    }

    QLineEdit::selection {
        background-color: #0078d7;
        color: #ffffff;
    }</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string extracomment="password"/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createAccountButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>480</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    font-family: Segoe UI;
    font-size: 18pt;
    font-weight: bold;
    color: #ffffff;
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
    border-radius: 8px;
    border-style: solid;
    border-width: 1px;
    border-color: #808080;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #4d4d4d;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 1 #2e2e2e);
}

QPushButton:pressed {
    background-color: #2e2e2e;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2e2e2e, stop: 1 #1c1c1c);
}

QPushButton:!hover:!pressed {
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
}</string>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QLabel" name="createAccountErrorLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>600</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginScreen">
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ArcaneCrypt</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>971</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Login to your account to view the contents of your Vaults</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginInput">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>310</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        font-family: Segoe UI;
        font-size: 9pt;
        border-radius: 4px;
    }
    
    QLineEdit {
        background-color: #ffffff;
    }
    
    QLineEdit {
        color: #000000;
    }
    
    QLineEdit {
        border-style: solid;
        border-width: 1px;
        border-color: #c9c9c9;
    }
    
    QLineEdit {
        padding-left: 4px;
        padding-right: 4px;
    }

    QLineEdit:hover, QLineEdit:focus {
        background-color: #f2f2f2;
    }

    QLineEdit::selection {
        background-color: #0078d7;
        color: #ffffff;
    }</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string extracomment="password"/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>480</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    font-family: Segoe UI;
    font-size: 18pt;
    font-weight: bold;
    color: #ffffff;
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
    border-radius: 8px;
    border-style: solid;
    border-width: 1px;
    border-color: #808080;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #4d4d4d;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 1 #2e2e2e);
}

QPushButton:pressed {
    background-color: #2e2e2e;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2e2e2e, stop: 1 #1c1c1c);
}

QPushButton:!hover:!pressed {
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginErrorLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>600</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainScreen">
     <widget class="QWidget" name="menuWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QComboBox" name="vaultList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Set the font family and size */
    QComboBox {
        font-family: Segoe UI;
        font-size: 9pt;
        font-weight: bold;
        color: #000000;
        border-radius: 4px;
        padding: 2px 20px 2px 6px;
        border: 1px solid #c9c9c9;
        background-color: #ffffff;
        selection-background-color: #0078d7;
        selection-color: #ffffff;
    }
    
    /* Set the background color and gradient */
    QComboBox::drop-down {
        background-color: #f2f2f2;
        border: 1px solid #c9c9c9;
        border-top-right-radius: 4px;
        border-bottom-right-radius: 4px;
        width: 20px;
    }
    
    /* Set the hover and focus background color */
    QComboBox:hover, QComboBox:focus {
        background-color: #f2f2f2;
        border: 1px solid #0078d7;
    }
    
    /* Set the background color and gradient of the dropdown menu */
    QComboBox QAbstractItemView {
        background-color: #ffffff;
        border: 1px solid #c9c9c9;
        selection-background-color: #0078d7;
        selection-color: #ffffff;
    }</string>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="createVaultButton">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>13</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font-family: Segoe UI;
    font-size: 12pt;
    font-weight: bold;
    color: #ffffff;
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
    border-radius: 8px;
    border-style: solid;
    border-width: 1px;
    border-color: #808080;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #4d4d4d;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 1 #2e2e2e);
}

QPushButton:pressed {
    background-color: #2e2e2e;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2e2e2e, stop: 1 #1c1c1c);
}

QPushButton:!hover:!pressed {
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
}</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addVaultButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>13</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font-family: Segoe UI;
    font-size: 12pt;
    font-weight: bold;
    color: #ffffff;
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
    border-radius: 8px;
    border-style: solid;
    border-width: 1px;
    border-color: #808080;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #4d4d4d;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 1 #2e2e2e);
}

QPushButton:pressed {
    background-color: #2e2e2e;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2e2e2e, stop: 1 #1c1c1c);
}

QPushButton:!hover:!pressed {
    background-color: #808080;
    background-image: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #808080, stop: 1 #4d4d4d);
}</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="vaultStack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>49</y>
        <width>971</width>
        <height>621</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>290</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Page1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>320</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>page2</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
